test/e2e/specs/all-features-confirmation/errors/errors.spec.js

```diff
@@ -75,65 +78,20 @@ describe('Error related test cases,', function() {
                   lastSlash = newapplink.lastIndexOf("/"),
                   recordsetUrl = newapplink.slice(0, lastSlash);
                 expect(currentUrl).toBe(recordsetUrl, "The redirection from record page to recordset/search in case of multiple records failed");
+                done();
             }).catch( function(err) {
```

Comment:
Since all these catch function will be the same you can just create a common function.

```javascript
var catchError =function (done) {
  return function (err) { 
   done.fail(err);
 }
}

it ("message", function (done) {
  promise.then(function () {

  }).catch(catchError(done))
});
```